quest/wavefunction.py

```diff
@@ -0,0 +1,35 @@
+"""
+A basic wave function class
+"""
+
+import numpy as np
+import psi4
+
+class WaveFunction(object):
+    """Basic wave function class"""
+
+    def __init__(self, psi4mol, basisname="sto-3g", scf_type="df",
+                 e_conv=1.e-8, d_conv=1.e-8):
+        psi4.set_options({'basis': basisname,
+                          'scf_type': scf_type,
+                          'e_convergence': e_conv,
+                          'd_convergence': d_conv})
+
+        bas = psi4.core.BasisSet.build(psi4mol, target=basis)
+
+        # Build the mints object
+        self.mints = psi4.core.MintsHelper(bas)
```

Comment:
As per out discussion, I think we want just a few fields `mints, arrays, variables, options`. We also want to take a Molecule class from Kate.